SignalRServiceBenchmarkPlugin/plugins/base/BenchmarkConfiguration/BaseBenchmarkConfiguration.cs

```diff
@@ -40,8 +40,7 @@ protected void Parse(string content)
             catch (Exception ex)
             {
                 Log.Error($"Benchmark configuration is invalid: {ex}");
-                throw new Exception("Benchmark configuration is invalid.");
-
+                throw;
```

Comment:
@vwxyzh 
Got it, thanks, I change most of the throw new Exception(message) to throw to obtain the whole exception object in catch block.

But convert the exceptions to string if the exception need to transfer from slaves to master, since Grpc doesn't support to pass exception from grpc server to grpc client.